grafana-tikv-dashboard.md

```diff
@@ -484,6 +484,45 @@ This section provides a detailed description of these key metrics on the **TiKV-
 - Initial Scanning Trigger Reason: The reason for triggering incremental scanning
 - Region Checkpoint Key Putting: The number of checkpoint operations logged to the PD
 
+### Snapshot restore 
+
+- Import CPU Utilization: CPU utilization aggregated by sst importer.
+- Import Thread Count: number of threads used by sst importer.
+- Import Errors: error counts during sst import.
+- Import RPC Duration: time spent for various RPC calls in sst importer.
+- Import RPC Ops: number of total RPC calls in sst importer.
+- Import RPC Count: number of inflight RPC calls in sst importer.
+- Import Write/Download RPC Duration: RPC time for write/download in sst importer.
+- Import Wait Duration: time spent on downloading task waiting in queue for execution.
+- Import Read SST Duration: time spent on reading from external storage of a file and download it to TiKV.
+- Import Rewrite SST Duration: time spent on rewriting SST based on rewrite rules. 
+- Import Ingest RPC Duration: time spent on handling ingest RPC request on TiKV.
+- Import Ingest SST Duration: time spent on ingesting SST into RocksDB.
+- Import Ingest SST Bytes: number of bytes ingested.
+- Import Download SST Throughput: SST download throughput in bytes per second.
+- TTL Expired: number of expired items after TTL in backup files.
+- cloud request: number of request to cloud providers. 
+
+### Point-in-Time Restore
+
+- CPU Usage: CPU utilization by PITR.
+- P99 RPC Duration: 99 percentile of the RPC requests time.
+- Import RPC Ops: number of total RPC calls in sst importer.
+- Import RPC Count: number of inflight RPC calls in sst importer.
```

Comment:
maybe move those 2 up

Reply:
oh, actually they are in both sections since PITR needs to download snapshot first and then download logs. 

I have raised this to the team that whether we should have a unified restore API instead of snapshot & PITR, the PITR in my opinion is a superset of the snapshot.